specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/examples/ManagedHsm_CreateOrUpdate.json

```diff
@@ -0,0 +1,88 @@
+{
+  "parameters": {
+    "resourceGroupName": "hsm-group",
+    "name": "hsm1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-04-01-preview",
+    "parameters": {
+      "properties": {
+        "tenantId": "00000000-0000-0000-0000-000000000000",
+        "initialAdminObjectIds": [
+          "00000000-0000-0000-0000-000000000000"
+        ],
+        "enableSoftDelete": true,
+        "softDeleteRetentionInDays": 90,
+        "enablePurgeProtection": true
+      },
```

Comment:
Can you please check if this is matching with the ManagedHsmProperties defined in the spec.

Reply:
I checked but could not find any differences except CreateMode string which is an optional parameter, so have not defined it in the request.
Please let me know if I am missing anything.

Reply:
I checked but could not find any differences except CreateMode string which is an optional parameter, so have not defined it in the request.
Please let me know if I am missing anything.